src/style.css

```diff
@@ -1,3 +1,72 @@
+html {
+  font-family: Roboto, sans-serif;
+}
+
 body {
   margin: 0;
+
+  --main-color: #00acdc;
+}
+
+.logo_img {
+  display: block;
+}
+
+.nav_link {
+  text-decoration: none;
+  position: relative;
+  color: black;
+  text-transform: uppercase;
+  height: 15px;
+  display: flex;
+  align-items: center;
+  font-size: 12px;
+  font-weight: 500;
+  padding: 22px 0;
+}
+
+.nav_link_margin {
+  margin-left: 20px;
```

Comment:
You should be consistent with your vertical margins. It's better to add only top or only bottom margin, but not both. In this case, you added a left margin, which is horizontal, but the rule still applies. It's better to add margins to one side (either left or right, but not both).